cebra/integrations/matplotlib.py

```diff
@@ -684,7 +684,7 @@ def _to_heatmap_format(
                 else:
                     heatmap_values[i, j] = score_dict[label_i, label_j]
 
-        return np.minimum(heatmap_values * 100, 99)
+        return np.minimum(heatmap_values * 100, 100)
```

Comment:
```suggestion
        return heatmap_values * 100
```

what about simply removing this in general?

Reply:
but otherwise also fine to merge